Dockerfile

```diff
@@ -48,8 +49,9 @@ RUN /bin/bash -c 'TMP_PKG_DIR=$(mktemp -d); \
     pip install --no-cache-dir $all_modules; \
     pip install --no-cache-dir --force-reinstall --upgrade azure-nspkg azure-mgmt-nspkg;'
 
-# Tab completion
-RUN cat /azure-cli/az.completion > ~/.bashrc
+# Tab completion and prompt
+RUN cat /azure-cli/az.completion > ~/.bashrc && \
+    echo -e '\n\nPS1="azure-cli# "' >> ~/.bashrc
```

Comment:
Can you remove this change?
I don't see what this adds from the default bash prompt.

Also, if for some reason a user does really want that prompt, they can do:
```
 docker run -it -e PS1="azure-cli# " azuresdk/azure-cli-python:2.0.20
```

Reply:
It's something I've done on my personal docker images so that I know what container prompt I'm at when running a number of containers in my terminal app.

Happy to take it out though. Will re-submit with this removed.